.s2i/bin/run

```diff
@@ -11,16 +11,19 @@
 # SOURCE and HOME DIRECTORY: /opt/app-root/src
 
 #set -x
+INVENTORY="tmp_inventory"
+INVENTORY_ARG="-i ${INVENTORY}"
```

Comment:
@aweiteka, I don't feel strongly about these refactorings I just needed some way to consistently reference the non-dynamic inventories.  I certainly didn't want to do `sed -i` on the inventory that was passed in.